lib/tapioca/compilers/dsl/url_helpers.rb

```diff
@@ -65,11 +65,13 @@ def generate_module_for(root, constant)
           end
         end
 
+        sig { params(mod: Parlour::RbiGenerator::Namespace, constant: T.class_of(Module), helper_module: Module).void }
         def create_mixins_for(mod, constant, helper_module)
-          mod.create_include(helper_module.name) if constant.ancestors.include?(helper_module)
-          mod.create_extend(helper_module.name) if constant.singleton_class.ancestors.include?(helper_module)
+          mod.create_include(T.must(helper_module.name)) if constant.ancestors.include?(helper_module)
```

Comment:
What does `T.must` do here?

Reply:
In general, the `name` method for a random `Module` could return `nil` if the module in question does not have a name. For example:
```ruby
a = Class.new
a.name # => nil
```
So the return type of the `.name` method is `T.nilable(String)`. But `create_include` is expecting a `String` which should not be `nil`. `T.must` tells Sorbet that we expect the return value of `helper_module.name` to **not** be `nil`. So Sorbet stops complaining. 

By the way, if code reaches this method, we **do** know that the module will definitely have a name, so this is safe to do.